Simplified/Book/Models/NYPLBookRegistryRecord.m

```diff
@@ -136,7 +161,14 @@ - (NSDictionary *)dictionaryRepresentation
   for (NYPLReadiumBookmark *readiumBookmark in self.readiumBookmarks) {
     [readiumBookmarks addObject:readiumBookmark.dictionaryRepresentation];
   }
-
+  
+  NSMutableArray *audiobookBookmarks = [NSMutableArray array];
+#if FEATURE_AUDIOBOOKS
+  for (NYPLAudiobookBookmark *audiobookBookmark in self.audiobookBookmarks) {
```

Comment:
this shouldn't be necessary, since `self.audiobookBookmarks` is a typed array at line 19.

Reply:
This transform the audiobook bookmark object to a dictionary representation, it's not a check.

Reply:
oops sorry, I got confused by similar code elsewhere.